doc/shared/Changelog.rst

```diff
@@ -24,6 +24,11 @@ Changelog
 Changes to SUNDIALS in release X.Y.Z
 ====================================
 
+**Bug Fixes**
+
+The shared library version numbers for the oneMKL dense linear solver and
+the PETSc SNES nonlinear solver libraries have been corrected.
+
```

Comment:
This only needs to go in `RecentChanges.rst`, that file is included by the line below. 
```suggestion
```